Doppler.Sap.Job/DopplerCurrencyService/BaseExternalService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Net.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Doppler.Sap.Job.Service.DopplerCurrencyService
+{
+    public abstract class BaseExternalService
```

Comment:
This class has no responsibility, what do you think about removing it?

Reply:
Yes, is an abstract class that I used on another approach, It's unnecessary I'll remove

Reply:
Done